rules/aip0136/http_uri_suffix_test.go

```diff
@@ -39,14 +39,8 @@ func TestURISuffix(t *testing.T) {
 		{"ValidOneWord", "Translate", "/v3:translate", testutils.Problems{}},
 		{"ValidStdMethod", "GetBook", "/v1/{name=publishers/*/books/*}", testutils.Problems{}},
 		{"ValidTwoWordNoun", "WriteAudioBook", "/v1/{name=publishers/*/audioBooks/*}:write", testutils.Problems{}},
-		{"ValidListRevisions", "ListBookRevisions", "/v1/{name=publishers/*/books/*}:listRevisions", testutils.Problems{}},
```

Comment:
The [History](https://google.aip.dev/162#history) of resource revisions indicates these _used_ to be custom methods, hinting at the fact that they are no longer.

Reply:
Right, AIP-162 is being reworked...technically there are still instances of these methods, but new ones are no longer allowed. Let's roll with this for now, I think they will still pass most custom method rules anyways.